tox.ini

```diff
@@ -4,7 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27,py33,py34,py35,py36,py37
+envlist = py36,py37,py38
```

Comment:
python-3.8 won't work yet; we need to figure out and merge #219 for that.  This PR will cause a merge conflict with that one, but it should be easily sorted once this merges.  Since we don't use tox ourselves, this is probably fine.